core/src/test/java/org/apache/druid/math/expr/FunctionTest.java

```diff
@@ -100,6 +105,7 @@ public void testStrlen()
   {
     assertExpr("strlen(x)", 3L);
     assertExpr("strlen(nonexistent)", NullHandling.defaultLongValue());
+    assertExprFail("strlen(a)", AssertionError.class, null);
```

Comment:
This test should be adjusted since you removed the strlen change.

Reply:
Removed this test for now. We can add it back when we fix https://github.com/apache/druid/issues/9824.